### ISSUE TITLE
Ignore stack traces coming from Cypress commands

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#453]
Capturing and printing stack traces coming from Cypress is essentially useless. It's always the same series of async failures and providing it creates confusion and indirection.

We should probably say in plain english: "Hey dev, this test failed due to a cypress command. We're omitting the stack trace".

Therefore when a test fails due to client code such as errors in your javascript code we can say: "Hey dev here's the stack trace because this came from your code. You need to fix it, kthx."

**Comment by jennifer-shehane** [comment]
We made a lot of improvements to error messages since this was opened. Closing as stale.
